lib/firewall.go

```diff
@@ -198,7 +200,7 @@ func (c *Client) GetFirewallRules(groupID string) ([]FirewallRule, error) {
 // protocol must be one of: "icmp", "tcp", "udp", "gre"
 // port can be a port number or colon separated port range (TCP/UDP only)
 func (c *Client) CreateFirewallRule(groupID, protocol, port string,
-	network *net.IPNet) (int, error) {
+	network *net.IPNet, notes string) (int, error) {
```

Comment:
After reading https://github.com/JamesClonk/vultr/issues/40 I think we should maybe create a `CreateFirewallRuleWithNote` and leave the method signature here untouched. What do you think?

Reply:
No, I think it's fine this way. We'll just bump the major version number since its a breaking aPI change, but not really a big deal. Anyone using external golang code as a library should vendor his dependencies imho, and thus not have any problem with such changes. :smile: